homeassistant/components/zha/core/discovery.py

```diff
@@ -182,59 +183,48 @@ def discover_group_entities(self, group: zha_typing.ZhaGroupType) -> None:
             )
             return
 
-        if group.entity_domain is None:
-            _LOGGER.debug(
-                "Group: %s:0x%04x has no user set entity domain - attempting entity domain discovery",
-                group.name,
-                group.group_id,
-            )
-            group.entity_domain = GroupProbe.determine_default_entity_domain(
-                self._hass, group
-            )
+        entity_domains = GroupProbe.determine_entity_domains(self._hass, group)
 
-        if group.entity_domain is None:
+        if len(entity_domains) < 1:
```

Comment:
```suggestion
        if not entity_domains:
```
?

Reply:
Damn, thought I fixed all of those 🤦🏻‍♂️